### PR TITLE
feat: Added highLight option to the action menu.

### PR DESCRIPTION
**Pull request by barath46**
## What does this change do?
- Added highlight option to the action menu.

### CODE REVIEW

File: src/display/editor/annotation_editor_layer.js [hunk 1]
```diff
@@ -809,8 +812,18 @@ class AnnotationEditorLayer {
   }
 
   createAnnotationNode() {
-    const { event, data } = this.#highlightSelectionParams;
-    this.#createAndAddNewEditor(event, false, data);
+    const editorType = this.#currentEditorType;
+    if (editorType) {
+      const { event, data } = this.#highlightSelectionParams;
+      this.#createAndAddNewEditor(event, false, data);
+      this.#highlightSelectionParams = null;
```
Comment (gpalsingh) [inline src/display/editor/annotation_editor_layer.js]: Isn't this already handled in this.removeTempHighlight()

File: src/display/editor/annotation_editor_layer.js [hunk 2]
```diff
@@ -809,8 +812,18 @@ class AnnotationEditorLayer {
...
+    } else {
+      const editor = new HighlightEditor.prototype.constructor({
+        ...this.#linkNodeParams,
+      });
+      this.add(editor);
+      this.#linkNodeParams = null;
```
Comment (gpalsingh) [inline src/display/editor/annotation_editor_layer.js:824]: Same here

File: src/display/editor/annotation_editor_layer.js (hunk 1 above)
Comment (barath46) [reply src/display/editor/annotation_editor_layer.js]: Yes bro, this is handled. I will remove from here bro

File: src/display/editor/annotation_editor_layer.js (hunk 2 above)
Comment (barath46) [reply src/display/editor/annotation_editor_layer.js:824]: But this one is not added to this.removeTempHighlight(). Do I need to add this also?

Comment (gpalsingh) [reply src/display/editor/annotation_editor_layer.js:824]: No bro, you are right, leave it as is

**Review by gpalsingh — APPROVED** [review]

**Pull request merged**